features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/NewOverview/Configuration.jsx

```diff
@@ -448,6 +448,31 @@ function Configuration(props) {
                                 {api.visibility === 'RESTRICTED' && api.visibleRoles.join()}
                                 {api.visibility === 'RESTRICTED' && ' ) '}
                             </Typography>
+                            {/* CORS Configuration */}
+                            <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
+                                <FormattedMessage
+                                    id='Apis.Details.NewOverview.Configuration.cors.configuration'
+                                    defaultMessage='CORS Configuration'
+                                />
+                            </Typography>
+                            <Typography component='p' variant='body1'>
+                                {api.corsConfiguration.corsConfigurationEnabled && (
+                                    <React.Fragment>
+                                        <FormattedMessage
+                                            id='Apis.Details.NewOverview.Configuration.cors.enabled'
+                                            defaultMessage='ENABLED'
+                                        />
+                                    </React.Fragment>
+                                )}
+                                {!api.corsConfiguration.corsConfigurationEnabled && (
+                                    <React.Fragment>
```

Comment:
I think we don't need a React Fragment here ?

Reply:
FIxed with 175b2ab5895ef33fb427396d3364f2062b8a385d